internal/database/migration/shared/data/cmd/generator/main.go

```diff
@@ -41,7 +41,7 @@ func mainErr() error {
 	for _, version := range versions {
 		versionTags = append(versionTags, version.GitTag())
 	}
-	fmt.Println(fmt.Sprintf("Generating stitched migration files for range [%s, %s]", MinVersion, MaxVersion))
+	fmt.Printf("Generating stitched migration files for range [%s, %s]", MinVersion, MaxVersion)
```

Comment:
```suggestion
	fmt.Printf("Generating stitched migration files for range [%s, %s]\n", MinVersion, MaxVersion)
```
think you dropped the newline? 🤔

Reply:
Thanks for catching that.